unison-syntax/src/Unison/Syntax/Parser.hs

```diff
@@ -371,7 +371,7 @@ tupleOrParenthesized :: Ord v => P v a -> (Ann -> a) -> (a -> a -> a) -> P v a
 tupleOrParenthesized p unit pair = seq' "(" go p
   where
     go _ [t] = t
-    go a xs = foldr pair (unit a) xs
+    go _ xs = foldr pair (unit External) xs
```

Comment:
This is just the final implicit "unit" that appears at the end of tuple pairs, it doesn't exist within the file, so shouldn't get the annotation of the full tuple.